actors/builtin/miner/miner_actor.go

```diff
@@ -1124,17 +1121,15 @@ func (a Actor) DeclareFaults(rt Runtime, params *DeclareFaultsParams) *adt.Empty
 			newFaultyPower, err := deadline.DeclareFaults(store, sectors, info.SectorSize, quant, faultExpirationEpoch, partitionsByDeadline[dlIdx])
 			builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to declare faults for deadline %d", dlIdx)
 
-			newFaultPowerTotal = newFaultPowerTotal.Add(newFaultyPower)
-
 			err = deadlines.UpdateDeadline(store, dlIdx, deadline)
 			builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to store deadline %d partitions", dlIdx)
+
+			newFaultPowerTotal = newFaultPowerTotal.Add(newFaultyPower)
```

Comment:
Do we use this? I don't think we need to return it from `dl.DeclareFaults` either.

Reply:
Yes, it's used below for `requestUpdatePower` to inform the power actor of the change.